src/main/java/br/edu/ufcg/computacao/eureca/backend/api/http/response/TeachersSummaryItemResponse.java

```diff
@@ -9,17 +9,19 @@
     private double averageFailDueToGrade;
     private double averageFailDueToAbsences;
     private double lockingRate;
+    private double successRate;
     private int totalEnrollments;
     private String from;
     private String to;
     private Glossary glossary;
 
     public TeachersSummaryItemResponse(String teacherName, double averageFailDueToGrade, double averageFailDueToAbsences,
-                                       double lockingRate, int totalEnrollments, String from, String to, Glossary glossary) {
+                                       double lockingRate, double successRate, int totalEnrollments, String from, String to, Glossary glossary) {
```

Comment:
Locking sounds odd. I used suspended in other places. It is not accurate either, but it is better than locking. I suggest use suspendedRate. If we find a better word latter, we will chance everywhere suspended by this new word.

Reply:
Done